crates/router/src/connector/worldline.rs

```diff
@@ -651,27 +654,148 @@ impl ConnectorIntegration<api::RSync, types::RefundsData, types::RefundsResponse
     }
 }
 
+fn is_endpoint_verification(headers: &actix_web::http::header::HeaderMap) -> bool {
+    headers
+        .get("x-gcs-webhooks-endpoint-verification")
```

Comment:
We don't receive any even type for endpoint verification in webhook, it can be differentiated only by checking the header "x-gcs-webhooks-endpoint-verification"